frontend/src/components/ScanForm/ScanForm.tsx

```diff
@@ -250,10 +250,10 @@ export const ScanForm: React.FC<{
             id="frequencyUnit"
             name="frequencyUnit"
             onChange={onTextChange}
-            value={values.frequencyUnit}
+            defaultValue={"day" || values.frequencyUnit}
```

Comment:
Let's not change it here, let's change it here instead (https://github.com/cisagov/crossfeed/blob/9464d2e924eead02433662c3a777b14351b5a3e4/frontend/src/components/ScanForm/ScanForm.tsx#L51)

Reply:
Good catch. Fixed.